content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<ul><li>Can you add the image that should be shown in the first cell of the notebook to the repo?</li><li>Is the first reference in the "What you'll do" section supposed to have a link, like the second one? Also, this paragraph has a trailing "u".</li><li>I believe there is an extra backslash in <code>*.npz</code></li><li> It would be nice to add links to the sites for matplotlib (https://matplotlib.org) and statsmodels (https://statsmodels.org)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Great ideas. Thanks for catching the typos
Should images be located in the "images" folder for the repo?

Reply:
<p>I think the images for tutorials should maybe live inside the content repo, just to make sure they are reachable in binder (binder will set up only the content part of the repo). Not sure if this will change if the repo workflow changes.</p>